### PR TITLE
docs: add link to quality assurance tables to Swagger UI

### LINKED ISSUE #762: Add link to quality assurance tables to Swagger UI docs.
## Description
As a contributor,
I need to know about test cases for each feature implemented on the backend.
so that I can easily improve code by creating issues, PRs, etc.

You could add the link to the quality assurance test cases found [here](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md). So that contributors know about test cases each feature failed or passed.

#### Note: This issue is labeled for Open Source Hack.
I can assign this issue to you on September 1st as the official opening date for Open Source Hack. Please free to check out other issues in the meantime. Program website: https://anitab-org.github.io/events/open-source-hack/

## Mocks
Currently, it looks like this:
mentorship-backend-2

### Update [Required]
- [ ] Add link to quality assurance tables.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[1] hour

### PR DESCRIPTION
**Pull request by anishghimire603**
### Description
I have added link to quality assurance tables to Swagger UI docs.

Fixes #762 

### Type of Change:

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

After adding changes, it looks like this:
image

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Update Swagger documentation and the exported file at /docs folder

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by paritoshsinghrahar** [comment]
@anishghimire603 I reviewed it. It would be great if you can add a brief description to what are quality assurance tables.

**Comment by anishghimire603** [comment]
> @anishghimire603 I reviewed it. It would be great if you can add a brief description to what are quality assurance tables.

Where should I add that in Swagger UI docs or in documentation or in the commit description?

**Comment by paritoshsinghrahar** [comment]
> Where should I add that in Swagger UI docs or in the documentation or in the commit description?

Swagger UI docs because the intention is to tell the contributors about test cases for each feature implemented on the backend. And to a new contributors, they need to know what quality assurance tables are.

**Comment by anishghimire603** [comment]
> > Where should I add that in Swagger UI docs or in the documentation or in the commit description?
> 
> Swagger UI docs because the intention is to tell the contributors about test cases for each feature implemented on the backend. And to a new contributors, they need to know what quality assurance tables are.

Can I take it from here or I have to research about it? And I have to add it in same line where i have added the link to QA table? 
https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md

**Comment by paritoshsinghrahar** [comment]
> > > Where should I add that in Swagger UI docs or in the documentation or in the commit description?
> > 
> > 
> > Swagger UI docs because the intention is to tell the contributors about test cases for each feature implemented on the backend. And to a new contributors, they need to know what quality assurance tables are.
> 
> Can I take it from here or I have to research about it? And I have to add it in same line where i have added the link to QA table?
> https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md

Yes, you can take this information. Keep it brief 1-2 lines at max.

**Comment by anishghimire603** [comment]
@paritoshsinghrahar 
image is this description okay?? I already committed this change.

**Comment by anishghimire603** [comment]
@paritoshsinghrahar any update require in my PR?

**Comment by paritoshsinghrahar** [comment]
Seems good for now. @anishghimire603

**Comment by anishghimire603** [comment]
> Seems good for now. @anishghimire603

will it be merged now?

**Comment by paritoshsinghrahar** [comment]
Yes, it will be merged by end of September. You can continue on to other issues. @anishghimire603

File: app/api/api_extension.py [hunk 1]
```diff
@@ -8,7 +8,10 @@
     + "through 1:1 relations during a certain period of time. \n \n"
     + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"
     + "The Android client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-android \n \n"
-    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"
+    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki \n \n"
+    + "<b>Quality Assurance Tables:</b>  Quality Assurance Tables are table containing Test Case and its Outcome as <i>Sucess</i> or <i>Fail</i>, the intention is to tell the contributors about test cases for each feature implemented on the backend. \n \n"
+    + "The link to the Quality Assurance Test cases found <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>here</a>."
```
Comment (isabelcosta) [inline app/api/api_extension.py]:
```suggestion
    + "This <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test scenarios evaluate if the API is working as it should."
```
I am not sure but line 12 is not showing up on your PR image 🤔 so I would frame it that way. What do you think? If you use this suggestion please delete line 12 :)

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@anishghimire603 thank you for this contribution 🎉 🤗 
I just left a minor component to improve the text that shows in the Swagger. Once you change it please update the image you have on the pull request description.

File: app/api/api_extension.py (hunk 1 above)
Comment (anishghimire603) [reply app/api/api_extension.py]: Sure, let me fix it and update my PR.

Comment (anishghimire603) [reply app/api/api_extension.py]: @isabelcosta I forget and wrote commit message as same as issue(add link to quality assurance tables to Swagger UI) title is should be like this (added link to quality assurance tables to Swagger UI) is that all right?

**Review by isabelcosta — DISMISSED** [review]
Looks good @anishghimire603 :)
Thank you for your contribution 🎉

File: app/api/api_extension.py (hunk 1 above)
Comment (isabelcosta) [reply app/api/api_extension.py]: It's ok :) Just keep in mind the Commit Message Guidelines you can find on our contribution guidelines.

**Comment by anishghimire603** [comment]
> Looks good @anishghimire603 :)
> Thank you for your contribution 🎉

welcome @isabelcosta looking to contribute in documentation part now to  be eligible for OSH.

**Comment by anishghimire603** [comment]
@SanketDG & @gaurivn it would be great if your review my changes 😊.

**Review by gaurivn — DISMISSED** [review]
Good, thank you :clap:

File: app/api/api_extension.py [hunk 2]
```diff
@@ -8,7 +8,8 @@
     + "through 1:1 relations during a certain period of time. \n \n"
     + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"
     + "The Android client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-android \n \n"
-    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"
+    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki \n \n"
+    + "This <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test scenarios evaluate if the API is working as it should."
```
Comment (SanketDG) [inline app/api/api_extension.py]:
```suggestion
    + "This <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test scenarios to evaluate if the API is working as it should."
```

**Review by SanketDG — CHANGES_REQUESTED** [review]
A small nitpick

File: app/api/api_extension.py (hunk 2 above)
Comment (anishghimire603) [reply app/api/api_extension.py]: sure let me fix this @SanketDG

**Comment by anishghimire603** [comment]
@SanketDG I have made those changes have a look.

**Review by SanketDG — APPROVED** [review]

**Comment by anishghimire603** [comment]
@gaurivn & @isabelcosta can you please again review my PR I had to make some changes to the Issue and your review got dismissed. 😁😁

**Review by isabelcosta — APPROVED** [review]
Looks good @anishghimire603 🎉 
Thank you for your contribution!

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Quality Assurance test cases link appears on Swagger UI_  
    _Screenshot/gif_:  
image
    _Expected Result_:  Quality Assurance test cases link appears on Swagger UI
    _Actual Result_: Quality Assurance test cases link appears on Swagger UI
    
3. Additional testcases covered: N/A 
4. Additional Comments: N/A

5. Status of PR Changed to: Ready to Merge.

**Comment by anishghimire603** [comment]
Had new and awesome experience working on this Issue, Thank you everyone from @anitab-org for the guidance. ❤️

**Pull request merged**